docs/assistant/use-attack-discovery-ai-assistant-incident-reporting.asciidoc

```diff
@@ -0,0 +1,65 @@
+
+In the example above, Attack discovery found connections between eleven alerts, and used them to identify and describe an attack chain.
+
+After Attack discovery outlines your threat landscape, use Elastic AI Assistant to quickly analyze a threat in detail. 
+
+[discrete]
+[[use-case-incident-reporting-use-ai-assistant-to-analyze-a-threat]]
+== Use AI Assistant to analyze a threat
+
+From a discovery on the Attack discovery page, click **View in AI Assistant** to start a chat that includes the discovery as context. 
+
+AI Assistant can quickly compile essential data and provide suggestions to help you generate an incident report and plan an effective response. You can ask it to provide relevant data or answer questions, such as “How can I remediate this threat?” or “What {esql} query would isolate actions taken by this user?” 
+
+image::images/attck-disc-esql-query-gen-example.png[An AI Assistant dialogue in which the user asks for a purpose-built {esql} query,90%]
+
+The image above shows an {esql} query generated by AI Assistant in response to a user prompt. Learn more about <<esql-queries-assistant,using AI Assistant for {esql}>>.
+
+At any point in a conversation with AI Assistant, you can add data, narrative summaries, and other information from its responses to {elastic-sec}'s case management system to generate incident reports. 
+
+[discrete]
+[[use-case-incident-reporting-create-a-case-using-ai-assistant]]
+== Create a case using AI Assistant
+
+From the AI Assistant dialog window, click **Add to case** (image:images/icon-add-to-case.png[Add to case icon,19,16]) next to a message to add the information in that message to a case. If you add a message that contains an Attack discovery to a case, AI Assistant automatically adds the attack summary and all associated alerts to the case. You can also add AI Assistant messages that contain remediation steps and relevant data to the case. 
+
+Cases help centralize relevant details in one place for easy sharing with stakeholders. Learn more about <<cases-overview,Cases>>. 
+
+[discrete]
+[[use-case-incident-reporting-translate]]
+== Translate incident information to a different human language using AI Assistant
+AI Assistant can translate its findings into other human languages, helping to enable collaboration among global security teams and distributed Security Operations Centers (SOCs), and making it easier to operate within multilingual organizations. 
+
+After AI Assistant provides information in one language, you can ask it to translate its responses. For example, if it provides remediation steps for an incident, you can instruct it to “Translate these remediation steps into Japanese.” You can then add the translated output to a case. This can help team members receive the same information and insights regardless of their primary language.
+
+NOTE: In our internal testing, AI Assistant translations preserved the accuracy of the original content. 
```

Comment:
Hm, I was actually expecting the opposite here: we're usually quite explicit about telling customers to take AI results with a grain of salt (for good reason!). Should we temper this note with the same healthy skepticism toward AI in general?